src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.cpp

```diff
@@ -336,7 +337,7 @@ void damped_harmonic_h(
     const double t_start_h_init, const double sigma_t_h_init,
     const double t_start_L1, const double sigma_t_L1, const double t_start_L2,
     const double sigma_t_L2, const double t_start_S, const double sigma_t_S,
-    const double sigma_r) noexcept {
+    const double sigma_r, const bool use_spec_style_rollon) noexcept {
```

Comment:
Do you want this to be an input file option or a compile time option?
For a compile time option make this a template parameter please.

Reply:
An input file option would be better I think, so I've added one now.